man/neptune_add.Rd

```diff
@@ -0,0 +1,43 @@
+\name{neptune_add}
+\alias{neptune_add}
+\title{
+Adds the provided tag or tags to the run's tags.
+}
+\description{
+Adds the provided tag or tags to the run's tags.
+}
+\usage{
+neptune_add(x, values, wait=FALSE)
+}
+\arguments{
+  \item{x}{Neptune handler, for which we want to add tags eg. run['sys/tags']}
```

Comment:
Neptune handler, for which we want to add tags eg. run['sys/tags']
->
Field for which we want to add tags eg. run['sys/tags']
Let's apply this change to all other methods as well.